dynawo/3rdParty/CMakeLists.txt

```diff
@@ -538,6 +538,45 @@ else()
   set(SUITESPARSE_HOME ${install_dir})
 endif()
 
+if(UNIX)
```

Comment:
agree they are different topic but this part is actually the only new piece of code regarding nicslu and all cmake are working so separating them would not be very productive

Reply:
I disagree with this as:
a. You will need to test compilation and trace generation again anyway due to the addition of a local copy of IODicos and Trace getter handles;
b. This commit is quite big as it is a melting pot of many little fixes, fix of coding style and two bigger modifications (fix of the bug of the trace and nicslu restoration). There is no ticket to trace most of those modifications, which is not a big deal when it is not affecting the functionalities, but it is not the case of nicslu fix. In addition, nicslu fix is not clearly mentioned in the commit message, meaning that there is no trace of this correction neither in the ticket system nor in the commit history. This is not good for maintenance and transparency;
c. I just double-checked and the multiple new pieces of code related to nicslu are easy to identify (always surrounded by a test on NICSLU_FOUND) and independant from the rest. I am not asking to filter the others modifications in the cmakes (in particular the sundials ones), I just would like to have an independant commit with only the nicslu related stuff so that it is easy to identify and visible. 
i hope this convinces you, otherwise I am available to discuss this anytime.